build/clean_build.sh

```diff
@@ -63,6 +63,7 @@ fi
 cd "$tc_dir"
 if [ -d "${tc_volume}/.git" ]; then
 	# In case the mirrored repo already exists, remove gitignored files
+	git config --global --add safe.directory '*'
```

Comment:
Adding a comment explaining the purpose of the `git` command would be nice, but not required :)